willywax/NextProduct
# #167944569 User can view a single product

#### What does this PR do?
Add a feature for the user to view a single product.
#### Description of Task to be completed?
- Find a product in the database using its id.
- Return the product to the user.
#### How should this be manually tested?
- Clone this repo and checkout to this branch
- Run `npm run dev`
- Singup or Signin to get token
- Add product into the database
- Visit http://127.0.0.1:5000/api/v1/products/:id
- See if the product is returned
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#167944569 ](https://www.pivotaltracker.com/story/show/167944569)
#### Screenshots (if appropriate)
<img width="1124" alt="Screen Shot 2019-08-22 at 08 32 25" src="https://user-images.githubusercontent.com/26115429/63492651-211caf80-c4ba-11e9-9ff7-fae87df670f7.png">
<img width="1131" alt="Screen Shot 2019-08-22 at 08 32 53" src="https://user-images.githubusercontent.com/26115429/63492662-2974ea80-c4ba-11e9-8b3e-0d0b6bc1cb48.png">
